### PR TITLE
VIH-10390 Fix for error starting hearing with new linked participant users

### PR DESCRIPTION
**Pull request by oliver-scott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/VIH-10390

### Change description ###
When a new user is created, we need to call video api to update their username in the database.
Instead of calling the API to update the whole participant, call a new endpoint to just update the username. This way the linked participants won't be deleted from them not being present in the update request.

### CODE REVIEW

File: charts/vh-booking-queue-subscriber/values.dev.template.yaml [hunk 1]
```diff
@@ -3,13 +3,14 @@ java:
   image: '${IMAGE_NAME}'
   releaseNameOverride: ${RELEASE_NAME}
   environment:
-    QUEUENAME: booking
+    VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-627.dev.platform.hmcts.net/
```
Comment (oliver-scott) [inline charts/vh-booking-queue-subscriber/values.dev.template.yaml:6]: TODO revert before merge

File: BookingQueueSubscriber/BookingQueueSubscriber.Services/BookingQueueSubscriber.Services.csproj [hunk 2]
```diff
@@ -23,7 +23,7 @@
       </PackageReference>
       <PackageReference Include="TimeZoneConverter" Version="3.3.0" />
       <PackageReference Include="UserApi.Client" Version="1.45.2" />
-      <PackageReference Include="VideoApi.Client" Version="1.48.6" />
+      <PackageReference Include="VideoApi.Client" Version="1.50.1-pr-0627-0002" />
```
Comment (oliver-scott) [inline BookingQueueSubscriber/BookingQueueSubscriber.Services/BookingQueueSubscriber.Services.csproj:26]: TODO update after video api merge

**Review by marcogagliardi — APPROVED** [review]
👌🏽

**Review by aurinelle — APPROVED** [review]
reviewed and tested

**Review by aurinelle — APPROVED** [review]

**Review by marcogagliardi — APPROVED** [review]
👌🏽

**Pull request merged**